tests/units/Toolbox.php

```diff
@@ -909,14 +909,22 @@ protected function shortenNumbers() {
             'number'    => 1600000000000,
             'precision' => 1,
             'expected'  => '1.6T',
-         ],
+         ], [
+            'number'    => "14%",
+            'precision' => 1,
+            'expected'  => '14%',
+         ], [
+            'number'    => "test",
+            'precision' => 1,
+            'expected'  => 'test',
```

Comment:
Why does a "number" method should allow that?

Reply:
>Maybe the method name is not correct in this case, because it does more things than just shortening the number (it also generate the HTML output).
>
>If we test is_numeric before then we need to split the code in two and handle the HTML output in another method.

Reply:
Can be done, just not sure it's worth the hassle.
Tell me if I should do it.

Reply:
Ok, no need to change ;)